test/bin/ci_phase_iso_build.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 #
-# This script runs on the hypervisor, from the iso-build step.
+# This script runs on the build host to create all test artifacts.
 
 set -xeuo pipefail
+export PS4='+ $(date "+%T.%N") ${BASH_SOURCE#$HOME/}:$LINENO \011'
 
-SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-# Cannot use common.sh because virsh is not installed, but we only
-# need ROOTDIR to set up logging in this script.
-ROOTDIR="$(cd "${SCRIPTDIR}/../.." && pwd)"
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=test/bin/common.sh
+source "${SCRIPTDIR}/common.sh"
```

Comment:
```suggestion
# Cannot use common.sh because some dependencies may be missing,
# but we only need ROOTDIR at this time
ROOTDIR="$(cd "${SCRIPTDIR}/../.." && pwd)"
SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```